docs/resources/dsm_sobject.md

```diff
@@ -58,6 +60,8 @@ The following attributes are stored in the `dsm_sobject` resource block:
 * **key\_size**: The security object key size from Fortanix DSM (matches the key_size provided during creation)
 * **key\_ops**: The security object key permission from Fortanix DSM
   * Default is to allow all permissions except "EXPORT"
+* **rsa**: rsaOptions passed as a string (if "RSA” `obj_type` is specified). The string should match the "rsa" value in Post body while working with Fortanix Rest API. For example, 
+`rsa = "{\"encryption_policy\":[{\"padding\":{\"RAW_DECRYPT\":{}}},{\"padding\":{\"OAEP\":{\"mgf\":{\"mgf1\":{\"hash\":\"SHA1\"}}}}}],\"signature_policy\":[{\"padding\":{\"PKCS1_V15\":{}}},{\"padding\":{\"PSS\":{\"mgf\":{\"mgf1\":{\"hash\":\"SHA384\"}}}}}]}"`
```

Comment:
expecting inline json feels wrong. Isn't there a way to convert terraform to json?

Reply:
Actually we tried a lot to address it as a map. Unfortunately we did not find any solution from terraform to read the nested map object. That's the reason we are expecting input as a string. 

In terrform list of map has some problem, go is not able to read the input. We did not find any solution on this.

Reply:
I assume you already looked at https://www.terraform.io/language/functions/jsonencode?